README.md

```diff
@@ -41,24 +40,26 @@ $ cd $GOPATH/src/github.com/ipfs/go-libp2p
 $ make
 ```
 
-# Examples
+## Usage
+
+`go-libp2p` repo will be a place holder for the list of Go modules that compose Go libp2p, as well as its entry point.
+
+## Examples
 
 Examples can be found on the [libp2p-examples repo](https://github.com/whyrusleeping/libp2p-examples/)
```

Comment:
This can be merged automatically, with #133. Any comments on this PR? @hsanjuan how does it look, otherwise?